### PR TITLE
Take GetStatusReport() & GetStatusReportEntryLength() of VPP back for…

### PR DESCRIPTION
**Pull request by pengche1**
… android

Change-Id: Ic806398c2bf47f76a46d2825223729b7ec8037df
Signed-off-by: peng.chen <peng.c.chen@intel.com>

### CODE REVIEW

**Comment by xuguangxin** [comment]
@XinfengZhang , please help review this

**Comment by xuguangxin** [comment]
@XinfengZhang , any update on this?
thanks

File: media_driver/agnostic/common/vp/hal/vphal.cpp [hunk 1]
```diff
@@ -789,11 +789,7 @@ MOS_STATUS VphalState::GetStatusReport(
             continue;
         }
 
-#if LINUX
```
Comment (Jister) [inline media_driver/agnostic/common/vp/hal/vphal.cpp]: Please keep the macro. If you want to enable it on Android, you can change it to:
#if (LINUX || ANDROID)

File: media_driver/agnostic/common/vp/hal/vphal.cpp [hunk 2]
```diff
@@ -816,11 +812,7 @@ MOS_STATUS VphalState::GetStatusReport(
         }
         else
         {   // here we have the first not ready entry.
-#if LINUX
```
Comment (Jister) [inline media_driver/agnostic/common/vp/hal/vphal.cpp]: Keep the Macro and change to: #if (LINUX || ANDROID)

Comment (pengche1) [reply media_driver/agnostic/common/vp/hal/vphal.cpp]: Sure. Thanks

**Pull request merged**